langkit/compiled_types.py

```diff
@@ -4234,7 +4234,7 @@ def __init__(self):
         ])
 
     def to_public_expr(self, internal_expr):
-        return 'To_Unbounded_Text (Image ({}))'.format(internal_expr)
+        return 'To_Unbounded_Text (Text_Type\'(Image ({})))'.format(internal_expr)
```

Comment:
Can you replace the single quotes that are string delimiters into double quotes, to avoid the need for escaping?